README.md

```diff
@@ -40,15 +33,19 @@ aws:
   instance:
     count: 5
 ```
-5. Build the plugin
+3. Build the plugin
 ```
 $ mkdir plugins
+$ make
+```
+4. Get the sample manifest
+```
 $ git clone https://github.com/lyraproj/puppet-workflow.git
 $ cp lyraproj/puppet-workflow/puppet/testdata/attach.pp lyraproj/lyra/plugins/
```

Comment:
Looks like these two steps are rendered moot by the addition of the file `plugins/attach.pp`?